constraint/pkg/client/client.go

```diff
@@ -377,6 +382,51 @@ func (c *Client) AddConstraint(ctx context.Context, constraint *unstructured.Uns
 	return resp, nil
 }
 
+// withDefaultParams injects any default values defined in the OpenAPIV3Schema
+// to the constraint spec only if a parameter is not already defined in the constraint.
+//
+//	Returns a copy of the constraint with the defaults or an error in processing the schema and constraint.
+func (c *Client) withDefaultParams(constraint *unstructured.Unstructured, templ *templates.ConstraintTemplate) (*unstructured.Unstructured, error) {
+	if templ.Spec.CRD.Spec.Validation == nil || templ.Spec.CRD.Spec.Validation.LegacySchema != nil {
```

Comment:
Why are we bypassing if legacySchema is defined?

Reply:
Good catch! This should be `OpenAPIV3Schema`.  I think early when I looked at this I was unsure what the relationship b/w `LegacySchema` and the actual openapiv3 spec was and figured they may be mutually exclusive! I know that's not the case now.

This line should actually make sure we have `Validation` and `OpenAPIV3Schema` available so we don't panic